From d2a823ef01396cd148074a99ccbbf1ca1115b24d Mon Sep 17 00:00:00 2001
From: Bernd Kuhls <bernd@kuhls.net>
Date: Sat, 3 Jan 2026 14:56:38 +0100
Subject: [PATCH] CMakeLists: fix build with cmake 4

Using CMake 4 the build is broken:

CMake Error at CMakeLists.txt:6 (cmake_minimum_required):
  Compatibility with CMake < 3.5 has been removed from CMake.

Upstream: https://github.com/microchip-ung/cfm/pull/1

Signed-off-by: Bernd Kuhls <bernd@kuhls.net>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5de2e0a..378e98d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -3,7 +3,7 @@
 
 project (cfm C)
 
-cmake_minimum_required(VERSION 2.6)
+cmake_minimum_required(VERSION 3.5)
 
 ## libnl ################################################
 find_path(LibNL_INCLUDE_DIR netlink/netlink.h
-- 
2.47.3

